lib/sidekiq/cli.rb

```diff
@@ -244,7 +244,7 @@ def boot_system
         end
         options[:tag] ||= default_tag
       else
-        require options[:require]
+        require options[:require] or raise ArgumentError, "#{options[:require]} was not required"
```

Comment:
The error message should probably be more helpful in fixing the problem: "sidekiq.rb was not required, you should use an explicit path: ./sidekiq.rb or /path/to/sidekiq.rb"